src/components/TextInput/BaseTextInput.js

```diff
@@ -72,13 +73,23 @@ class BaseTextInput extends Component {
         this.input.focus();
     }
 
-    componentDidUpdate(prevProps) {
+    componentDidUpdate(prevProps, prevState) {
         // Activate or deactivate the label when value is changed programmatically from outside
         const inputValue = _.isUndefined(this.props.value) ? this.input.value : this.props.value;
         if ((_.isUndefined(inputValue) || this.state.value === inputValue) && _.isEqual(prevProps.selection, this.props.selection)) {
             return;
         }
 
+        if (this.props.autoGrow) {
+            if (inputValue !== this.state.hiddenInputValue) {
+                this.setState({hiddenInputValue: inputValue, selection: this.props.selection});
+            }
+
+            if (prevState.textInputWidth === this.state.textInputWidth && this.props.shouldWaitWidthCalculation) {
+                return;
```

Comment:
In this case, the entire function ends. Am I right?

Reply:
yup